chef/cookbooks/uwsgi/metadata.rb

```diff
@@ -0,0 +1,8 @@
+name            "uwsgi"
+maintainer       "Crowbar Project"
+maintainer_email "crowbar@googlegroups.com"
+license          "Apache 2.0"
```

Comment:
Style/ExtraSpacing: Unnecessary spacing detected.<br>Style/SingleSpaceBeforeFirstArg: Put one space between the method name and the first argument.